tidb-binlog/tidb-binlog-overview.md

```diff
@@ -54,11 +54,16 @@ The TiDB Binlog cluster is composed of Pump and Drainer.
 
 ## Notes
 
-* TiDB Binlog is not compatible with the following features introduced in TiDB v5.0 and they cannot be used together. It is recommended to use [TiCDC](/ticdc/ticdc-overview.md) instead of TiDB Binlog:
+* In v5.1, the incompatibility between introduced clustered index and TiDB Binlog has been solved. After you update TiDB Binlog and TiDB Server to v5.1 and enable TiDB Binlog, TiDB will support creating clustered indexes and data insertion, deletion, and update of the created clustered index tables will be replicated downstream via TiDB Binlog. Pay attention to the replication of clustered index tables:
 
-    - [TiDB Clustered Index](/clustered-indexes.md#limitations): After TiDB Binlog is enabled, TiDB does not allow creating clustered indexes with non-single integer columns as primary keys; data insertion, deletion, and update of the created clustered index tables will not be replicated downstream via TiDB Binlog.
-    - TiDB system variable [tidb_enable_async_commit](/system-variables.md#tidb_enable_async_commit-new-in-v50): After TiDB Binlog is enabled, performance cannot be improved by enabling this option.
-    - TiDB system variable [tidb_enable_1pc](/system-variables.md#tidb_enable_1pc-new-in-v50): After TiDB Binlog is enabled, performance cannot be improved by enabling this option.
+    - If you manually control the upgrade sequence to upgrade v5.0, make sure that TiDB binlog is upgraded to v5.1 before upgrading the TiDB server to v5.1.
```

Comment:
@lysu PTAL whether my comment is correct. Thanks!
```suggestion
    - If you have upgraded the cluster to v5.1 from v5.0 by manually controlling the upgrade sequence, make sure that TiDB binlog is upgraded to v5.1 before upgrading the TiDB server to v5.1.
```

Reply:
LGTM